rootfs/usr/local/bin/grafana-db

```diff
@@ -23,7 +23,7 @@ Usage:
 
 Example:
   # Load the latest version of kube-prometheus standard Kubernetes Grafana dashboards, replacing any already uploaded
-  grafana-db --overwrite https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/master/manifests/grafana-dashboardDefinitions.yaml
+  grafana-db --overwrite https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/main/manifests/grafana-dashboardDefinitions.yaml
 
   # Load the Kubernetes Capacity dashboard with gnetId 5228 from Grafana.com
   grafana-db 5228
```

Comment:
I think we should drop this script. We haven't used it in many years.

Reply:
We have seen renewed interest in Grafana in just the past few weeks. Let's leave this in for now and revisit in 3 months.